CSVSource/build.gradle

```
@@ -78,3 +78,31 @@ tasks.withType(Test) {
         systemProperty 'EntFullFilePath', rootProject.findProperty('EntFullFilePath')?:"b.csv"
     }
 }
+
+// Using gradle-dcompose-plugin to build CSVSource docker image
+dcompose {
+    if (rootProject.hasProperty('dockerRegistry')) {
+        def registryName = rootProject.getProperty('dockerRegistry')
+        csvSource {
+            buildFiles = project.copySpec {
+                from "$projectDir/src/main/docker/"
+                from "$projectDir/build/distributions/CSVSource.tar"
+            }
+            if (rootProject.hasProperty('csvImageTag')) {
```

Make the image tag generic.  If someone wants to publish everything with different tags, they can do 20 publishes.

```suggestion
            if (rootProject.hasProperty('imageTag')) {
```